cdap-common/bin/common.sh

```diff
@@ -167,6 +167,9 @@ cdap_set_hbase() {
       1.1*)
         hbasecompat="$CDAP_HOME/hbase-compat-1.1/lib/*"
         ;;
+      1.2-cdh*)
+        hbasecompat="$CDAP_HOME/hbase-compat-1.2-cdh5.7.0/lib/*"
```

Comment:
This is for HBase version 1.2.0-cdh5.7.0 which is different from Apache HBase 1.2. We currently do not have requirement to support Apache HBase 1.2 as its not supported by any disto yet.